proto/tendermint/abci/types.proto

```diff
@@ -277,6 +277,12 @@ message ResponseCheckTx {
   // removed).
   reserved 9 to 11;
   reserved "sender", "priority", "mempool_error";
+
+
+  // these fields are used in the priority mempool
+  bytes address = 15;
+  uint64 gas_fees = 16;
+  uint64 sequence = 17;
```

Comment:
will these new fields be used in another PR? or this PR is still not ready for review yet?

Reply:
the mempool in the code currently does not work with priority so there is nothing to add here

Reply:
I see, and when/where will they be used?

Reply:
in the cat mempool and if we decide to bring back priority mempool.

Reply:
cool 👍